tiledb/ml/_utils.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
These are testing utils, so please move them there.

Reply:
Please check the discussion [here]( https://github.com/TileDB-Inc/TileDB-ML/pull/49#discussion_r649020611)